go.mod

```diff
@@ -9,4 +9,5 @@ require (
 	github.com/jackc/pgx/v4 v4.11.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
+	go.mongodb.org/mongo-driver v1.7.3 // indirect
```

Comment:
I think the `// indirect` part will be removed if you run `go mod tidy`. This appears to be a direct dependency not an indirect dependency.

Reply:
Exactly.